prow/plank/plank_test.go

```diff
@@ -510,6 +516,7 @@ func TestSyncKubernetesJob(t *testing.T) {
 			expectedPodHasName: true,
 			expectedNumPods:    1,
 			expectedReport:     true,
+			expectedURL:        "NEWPOD/pending",
```

Comment:
wasn't sure of exactly what the issue should be so I created it and cc'ed you ! :)